docs/accessibility/core-concepts/comparing-runs.mdx

```diff
@@ -0,0 +1,110 @@
+
+<DocsImage
+  src="/img/accessibility/core-concepts/branch-review-example-diff.png"
+  alt="An example of a Cypress Accessibility report comparison in the Branch Review area. The example shows no new failed rules, no resolved rules, and one failed rule with changes - the 'form elements must have labels' rule, which has two new failures on the `/contact` route."
+/>
+
+
+## Use cases
+
+Comparing runs can be useful in various scenarios.
+
+**Key use cases:**
+
+- **Pre-merge checks**: Know if any net-new issues are introduce by UI code changes.
+- **Monitoring changes**: Compare nightly monitoring runs and track down the introduction of new problems caused by underlying changes in the application.
+- **Detecting content issues**: Sometimes content editors can introduce accessibility issues unrelated to code changes. Seeing the example issues presented visually, in context, helps you quickly triage whether you are dealing with a recent code change issue, or a content authorship problem.
+- **Reviewing AI-generated code changes**: The increased use of AI to generate and/or review front-end code creates some increased risks of accessibility regressions making it to production. Seeing the increase or decrease of accessibility issues when reviewing a pull request.
+
+
+## Content of the report
+
+<DocsImage
+  src="/img/accessibility/core-concepts/branch-review-intro.png"
+  alt="Cypress Accessibility Branch Review with arrows calling out the sections for new failed rules, resolved rules, and failed rules with changes. New failed rules are defined as rules that just started failing on the new branch. Resolved rules are defined as rules that got fully fixed in the new branch. Failed rules with changes are defined as rules that failed on both branches, but with different element counts or different related pages."
+/>
+
+
+The Branch Review report is prioritized into three sections:
+
+- **New failed rules**: Rules that were passing every time they ran on the `base` run but now have failures in the `changed` run.
+- **Resolved rules**: Rules that had some failures detected in the `base` run but have no failures on the `changed` run. This helps you celebrate the wins and recognize when a new run has gone green.
+- **Failed rules with changes**: Rules that were failing in both runs, where the elements with failures detected have increased, decreased, or changed in some way.
+
+This organization of the report brings the most significant results to the top - regressions of rules that had been fully passing in the `base` run. Increases or decreased in the element counts for rules that already have some failures provide a good sense of progress, but can tend to be noisier, especially if the runs have different content or other conditions that changed in between.
+
+## How to compare runs
```

Comment:
It's possible we should just move all of this to https://docs.cypress.io/cloud/features/branch-review which is a little bit stale and doesn't describe all the ways you can jump into a review. For a casual reader of the accessibility docs, I think understanding these entrypoints is really useful, so maybe the duplication is OK.